components/configuration-mgt/org.wso2.carbon.identity.configuration.mgt.core/src/test/resources/testng.xml

```diff
@@ -0,0 +1,26 @@
+<!--
+  ~ Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~ http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
+
+<suite name="ConfigurationManagementCoreTestSuite" object-factory="org.powermock.modules.testng.PowerMockObjectFactory">
+
+    <test name="ConfigurationManagementCoreTests" preserve-order="true" parallel="false">
+        <classes>
+            <class name="org.wso2.carbon.identity.configuration.mgt.core.ConfigurationManagerTest"/>
+        </classes>
+    </test>
+</suite>
```

Comment:
add new line EOF

Reply:
Fixed in d0a5b5ad8ff1bdab587b5bd0c8c0cc2c3b6d4afb